app_flutter/lib/widgets/task_icon.dart

```diff
@@ -44,22 +44,19 @@ class TaskIcon extends StatelessWidget {
           color: blendFilter,
         );
       case StageName.luci:
-        return Image.asset(
-          'assets/chromium.png',
-        );
-      case StageName.devicelab:
-        return Image.asset(
-          'assets/android.png',
-        );
-      case StageName.devicelabWin:
-        return Image.asset(
-          'assets/windows.png',
-        );
-      case StageName.devicelabIOs:
-        return Image.asset(
-          'assets/apple.png',
-          color: blendFilter,
-        );
+        if (qualifiedTask.builder.toLowerCase().startsWith('linux')) {
+          return Image.asset(
+            'assets/linux.png',
```

Comment:
Can you ensure all images have `color: blendFilter` to ensure dark mode support

Reply:
done